test/e2e/catalog_e2e_test.go

```diff
@@ -1309,6 +1309,181 @@ var _ = Describe("Starting CatalogSource e2e tests", func() {
 			}
 		}
 	})
+
+	When("A CatalogSource is created with a properly formed CSV", func() {
+		var (
+			og               operatorsv1.OperatorGroup
+			namespaceName    string
+			ns               corev1.Namespace
```

Comment:
You don't need another ns. There's a global one that gets created and torndown before every spec: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/test/e2e/catalog_e2e_test.go#L61